server/src/main/java/com/networknt/server/AppStatusStartupHook.java

```diff
@@ -0,0 +1,45 @@
+package com.networknt.server;
+
+import com.networknt.config.Config;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Map;
+
+/**
+ * The startup hook is used to merge status.yml and app-status.yml
+ * <p>
+ * 1. If repeated elements is contained by these two files, throw a RuntimeException when startup.
+ * 2. If elements contained by app-status.yml is new, append them to status.yml
+ * <p>
+ * To implement this startup hook, uncomment it in service.yml
+ *
+ * @author Jiachen Sun
+ */
+public class AppStatusStartupHook implements StartupHookProvider {
+    private static final Logger logger = LoggerFactory.getLogger(AppStatusStartupHook.class);
+
+    public static final String[] CONFIG_NAME = {"status", "app-status"};
+    public static Map<String, Object> config = Config.getInstance().getJsonMapConfig(CONFIG_NAME[0]);
+    public static final Map<String, Object> appStatusConfig = Config.getInstance().getJsonMapConfigNoCache(CONFIG_NAME[1]);
+
+    @Override
+    public void onStartup() {
+        if (appStatusConfig == null) {
+            return;
+        }
+        for (String key : appStatusConfig.keySet()) {
+            if (config.containsKey(key)) {
+                if (logger.isInfoEnabled()) {
+                    logger.error("The status code: " + key + " has already in use by light-4j and cannot be overwritten," +
+                            " please change to another status code in app-status.yml if necessary.");
+                }
+                System.out.println("The status code: " + key + " has already in use by light-4j and cannot be overwritten," +
```

Comment:
Hi @jiachen1120 ,
Do you use System.out.println on purpose here? I think logger above is enough, isn't it?

Reply:
Hey! Thanks for the advice! I will remove it and move the details of the error information to Exception. I am just wondering if it may be difficult to find the problem if the log is disabled. It may be more intuitive to print out it to make sure the developer can see it for the first time? Because this sentence will be very easy to find at the bottom of the console.